src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/table/table.component.scss

```diff
@@ -63,7 +63,7 @@
   }
   .input-group {
     float: right;
-    border-left: 1px solid rgba(0,0,0,.09);
+    border-left: $border-color-white;
```

Comment:
Should we extract the whole rule or just the color?
Do you think we will need to change the border size and texture when customizing the template?
If we extracted just the color we might be able to reduce the number of color variables.

Reply:
Ack, it is better to assign variable just to color.